app/controllers/application_controller/compare.rb

```diff
@@ -1208,15 +1208,17 @@ def drift_add_section_field_expanded(view, section, field)
       next if fld.nil?
       val = fld[:_value_].to_s
 
+      style = nil
       if idx.zero? # On base object
         img_bkg = "cell-stripe"
       elsif fld[:_match_]
         img_bkg = "cell-bkg-plain-no-shade"
       else
+        style = "color: #21a0ec;" if fld[:_match_] == false
```

Comment:
```ruby
... unless fld[:_match__]
```

Reply:
Here we want to set style ONLY when there is `_match_` key and if it is set to `false`. Often, `fld` looks like this: {`:_value_=>"(empty)"}`, and then if I used `unless fld[:_match_]`, style would be set, and we don't want this in this case. So I better used condition in this format: `if fld[:_match_] == false`, to make sure that the style is set really only when we want it. Fortunatelly, when `fld` has no `_match_` key, `idx` is usually zero so it will not enter the `else` part of the block and nothing bad should happen, in fact. But I don't want to rely on this (on presence of this line: https://github.com/ManageIQ/manageiq-ui-classic/pull/3938/files#diff-559128a3af015518ec2c6d248d03ef7aR1212).